api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -493,26 +496,35 @@ public void testSubmitDataUseAgreement_success() {
     assertThat(dbUserDataUseAgreement.getUserFamilyName()).isEqualTo(dbUser.getFamilyName());
     assertThat(dbUserDataUseAgreement.getUserGivenName()).isEqualTo(dbUser.getGivenName());
     assertThat(dbUserDataUseAgreement.getUserInitials()).isEqualTo(duaInitials);
-    assertThat(dbUserDataUseAgreement.getDataUseAgreementSignedVersion()).isEqualTo(DUA_VERSION);
+    assertThat(dbUserDataUseAgreement.getDataUseAgreementSignedVersion()).isEqualTo(DUCC_VERSION);
   }
 
   @Test
-  public void testSubmitDataUseAgreement_wrongVersion() {
+  public void testSubmitDataUseAgreement_wrongVersion_older() {
     assertThrows(
         BadRequestException.class,
         () -> {
           createAccountAndDbUserWithAffiliation();
           String duaInitials = "NIH";
-          profileController.submitDataUseAgreement(DUA_VERSION - 1, duaInitials);
+          profileController.submitDataUseAgreement(DUCC_VERSION - 1, duaInitials);
         });
   }
 
+  // not really a use case for this, but shows we need an exact match
+
   @Test
-  public void test_outdatedDataUseAgreement() {
```

Comment:
the new version of this test (just below) uses a mock/spy instead of a feature flag